x-pack/solutions/observability/plugins/slo/public/components/slo/slo_outdated_callout/index.tsx

```diff
@@ -11,18 +11,22 @@ import { FormattedMessage } from '@kbn/i18n-react';
 import { useKibana } from '../../../hooks/use_kibana';
 import { useFetchSloDefinitions } from '../../../hooks/use_fetch_slo_definitions';
 import { paths } from '../../../../common/locators/paths';
+import { useUrlSearchState } from '../../../pages/slo_management/components/hooks/use_url_search_state';
 
 export function SloOutdatedCallout() {
+  const { onStateChange } = useUrlSearchState();
+
   const {
     application: { navigateToUrl },
     http: { basePath },
   } = useKibana().services;
 
   const handleClick = () => {
-    navigateToUrl(basePath.prepend(paths.slosOutdatedDefinitions));
+    onStateChange({ includeOutdatedOnly: true });
+    navigateToUrl(basePath.prepend(paths.slosManagement));
```

Comment:
Does this work?  I guess it works because by calling this stateManager we update the session storage, and when we redirect to the management page, we read from the URL (nothing) and then the session storage (something)?

Reply:
I think we should just redirect to the slo management page with the url crafted for the isOutdated filter. I think we have some example in the `paths` object.